src/apps/virtio_net/virtio_net.lua

```diff
@@ -0,0 +1,76 @@
+-- Application to connect to a virtio-net driver implementation
+--
+-- Licensed under the Apache 2.0 license
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Copyright (c) 2015 Virtual Open Systems
```

Comment:
> I would like to avoid these license and copyright lines in the source files. To me it is boilerplate that doesn't add value. We already spell out the license in the COPYING file and Git already tracks the authorship very directly. If those mechanisms are not sufficient for some reason then it would be best to open an Issue to discuss that rather than starting to add random notes in the source code.

Absolutely agree, `git blame` should work for the people who really need to know who wrote a specific line of code. I get that people want to express their authorship but I don't think it makes sense to spell it out on a per-file basis. Once somebody unrelated changes a line in a “copyrighted file” the copyright notice becomes bogus, no? Ah, the beauty of open source... :-)